recipes/logzero/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "logzero" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "f3433f87a184e1d846099c521437bfe9d463d4b74c3b2da3488b4f52ce821a1d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - colorama
```

Comment:
`colorama` appears to be a windows-only requirement. If this is the case, please add the proper selector:

```
- colorama  # [win]
```

If that's added then you have to remove the `noarch: python`

Reply:
thanks, you're right. Fixing in a minute.